lib/httpadapter/fetchadapter.js

```diff
@@ -18,8 +18,8 @@ class FetchAdapter {
   get(url, params, callback, fullResponse = false) {
     var options = {
       headers: {
-        'user-agent':
-          'Mozilla/5.0 (X11; Linux i586; rv:31.0) Gecko/20100101 Firefox/31.0'
+        'user-agent': 'Mozilla/5.0 (X11; Linux i586; rv:31.0) Gecko/20100101 Firefox/31.0',
+        'accept': 'application/json;q=0.9, */*;q=0.1'
```

Comment:
Not sure about this, since will affect other providers but I didn't find a way to specify this header from `HereGeocoder` but if you don't, all responses from Here Batch API will be in XML.

Reply:
I think it should work I started to add some integration test so we can verify that it do not break every provider.